src/WCSimTuningParameters.cc

```diff
@@ -19,6 +19,9 @@ WCSimTuningParameters::WCSimTuningParameters()
  tvspacing = 100.0;
  topveto = false;
 
+ WCCladdingReflectivity   = 0.90; //
```

Comment:
According to the tuning parameter messenger, this is the OD WLS cladding (rather than any other cladding). Can you use a more descriptive name, such as
```suggestion
 WCODWLSCladdingReflectivity   = 0.90; //
```

Reply:
Done. See 12b24ff7c8d3984fb566ed476d87e0d474f29781